docs/pages/pmd/devdocs/pull_request_merge.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Pull-Request Merge
+permalink: pmd_devdocs_pull_request_merge.html
+last_updated: August 2017
+author: Andreas Dangel <andreas.dangel@adangel.org>
+---
+
+## Example 1: Merging PR #123 into master
+
+1.  Review
+
+    *   Compilation and checkstyle is checked already by travis build: PRs are automatically verified.
+    *   If it is a bug fix, a new unit test, that reproduces the bug, is mandatory.
+    *   Add the appropriate labels on the github issue: If the PR fixes a bug, the label "a:bug" should be used.
+    *   Make sure, the PR is added to the appropriate milestone.
+
+2.  The actual merge commands:
+
+    We assume, that the PR has been created from the master branch. If this is not the case,
+    then we'll either need to rebase or ask for rebasing before merging.
+
+    ```
+    git checkout master && git pull origin master
+    git checkout -b pr-123
+    git pull https://github.com/otheruser/pmd.git branchname
```

Comment:
currently I'm using `git fetch origin pull/<id>/head:pr-<id>` instead of these 2 last steps. That way I get the contents of the PR in an easier way, branched from the same point the submitted did, and have at most 1 merge over master at the end